src/shared/dom.js

```diff
@@ -22,6 +22,20 @@ export function teardownEach ( iterations, detach, start ) {
 	}
 }
 
+export function spreadAttributes ( node, obj ) {
+	if (obj !== null && typeof obj === 'object') {
+		for ( var property in obj ) {
```

Comment:
According to this Object.keys + for loop is faster: https://jsperf.com/object-keys-vs-for-in-with-closure/3